recipes/rnaz/meta.yaml

```diff
@@ -7,12 +7,17 @@ source:
   sha256: b32ec0361889319f2058f224d6c456c853dbc30dff4dba90c53a8f9fd7b83be5
 
 build:
-  number: 2
+  number: 3
   skip: True # [osx]
 
+patches:
+  - "rnaz.patch"
+  
 requirements:
   build:
     - {{ compiler('cxx') }}
+  host:
+    - libsvm =3.21
```

Comment:
Is the pinning needer here?

Reply:
I see it is, because it is hard coded in your patch.

Reply:
Yeah, has to be that way, its hard coded into RNAz, if you not provide the exact version it doesn't run, also the compiler options have to be set, but it would need a major overhaul of the source code to fix that